src/dmd/statementsem.d

```diff
@@ -3445,7 +3445,12 @@ private extern (C++) final class StatementSemanticVisitor : Visitor
             if (tbret.ty != Tvoid && !resType.isTypeNoreturn()) // if non-void return
             {
                 if (tbret.ty != Terror)
-                    rs.error("`return` expression expected");
+                {
+                    if (e0)
+                        rs.error("Expected return type of `%s`, not `%s`", tret.toChars(), resType.toChars());
```

Comment:
"Expected" should not be capitalized here

Reply:
Fixed